components/not-found/element.js

```diff
@@ -89,7 +89,7 @@ export class MDNNotFound extends L10nMixin(LitElement) {
                       </a>
                     </li>`,
                 )}
-              </ul></p>
+              </ul>
```

Comment:
Not sure I see the issue here: `<p><ul>...</ul></p>` is perfectly valid html, no? Was this just a manual change?

Reply:
As a workaround, I now extracted the list into a variable: bb359d3